src/Unleash.php

```diff
@@ -127,7 +127,7 @@ function () {
 
     protected function fetchFeatures(): array
     {
-        $response = $this->client->get($this->getFeaturesApiUrl(), $this->getRequestOptions());
+        $response = $this->client->get($this->config->get('unleash.featuresEndpoint'));
```

Comment:
@mikefrancis I think this would be a breaking change too, actually, if someone used this update without knowing to add `unleash.featuresEndpoint` to their existing config file.

Might be best if I restore the `getFeaturesApiUrl()` function?

Reply:
Ah, no. Not required, because of course you merge the solution's custom config with the package's config in your `register` method, which means that the default will be used even if the setting is left out of the custom config.

And I've tested and confirmed that theory.